src/components/Rules/index.js

```diff
@@ -1,61 +1,74 @@
 import { Rule, CloseButton } from './styled';
-import { IoMdCloseCircle } from "react-icons/io";
+import { IoMdCloseCircle } from 'react-icons/io';
 
-const Rules = ({setShown}) => {
-
-  return(
+const Rules = ({ setShown }) => {
+  return (
     <Rule id="modal">
       <h1>Dino-Battle!</h1>
```

Comment:
This one was my fault for overlooking it, but we should probably change the title to "Dino-War" instead to match with the home screen and tab. Or perhaps change it to just "Rules", "How-To", or something.

Reply:
Changed to Dino-War. I think its okay to have the title of the game at the top.